### PR TITLE
Fix dimensions params

### PR DESCRIPTION
**Pull request by emezac**
Fix dimensions params.

This fix is for: https://github.com/andreibondarev/langchainrb/issues/506.

Is related with the latest changes of https://openai.com/blog/new-embedding-models-and-api-updates

Issue **Before:**

Captura de pantalla 2024-03-05 a la(s) 9 35 45 a m

After: 
Captura de pantalla 2024-03-05 a la(s) 9 47 12 a m

### CODE REVIEW

**Comment by emezac** [comment]
@andreibondarev please take a look at my implementation and the proper test specs that I added.

File: lib/langchain/embeddings.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Langchain
+  module Config
+    EMBEDDING_SIZES = {
```
Comment (andreibondarev) [inline lib/langchain/embeddings.rb:5]: Do we intend to put ALL of the embedding sizes across the different LLM providers in 1 constant?

Comment (emezac) [reply lib/langchain/embeddings.rb:5]: I wasn't too sure about this constant and I don't know what OpenAI's evolution strategy is, but I intended to be able to have these constants available to the class if needed, as in the case of the test specification. I think this OpenAI's approach is only for embeddings

Comment (emezac) [reply lib/langchain/embeddings.rb:5]: Everybody is having queries/concerns about this changes -> : https://community.openai.com/t/how-to-deal-with-different-vector-dimensions-for-embeddings-and-search-with-pgvector/602141

Comment (emezac) [reply lib/langchain/embeddings.rb:5]: I added a screenshot of the current behavior and the fixed behavior.

Comment (emezac) [reply lib/langchain/embeddings.rb:5]: I ran all my tests yesterday  night and all passed, not sure why currently some are broken now

**Comment by andreibondarev** [comment]
@emezac Would you please take a look at the spec failures? They look like actual bugs.

**Comment by emezac** [comment]
> @emezac Would you please take a look at the spec failures? They look like actual bugs.

Yeap I will take care of it.

File: lib/langchain/llm/openai.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ class OpenAI < Base
       n: 1,
       temperature: 0.0,
       chat_completion_model_name: "gpt-3.5-turbo",
-      embeddings_model_name: "text-embedding-ada-002"
+      embeddings_model_name: "text-embedding-3-small"
```
Comment (andreibondarev) [inline lib/langchain/llm/openai.rb:20]: @emezac I modified the default model per this suggestion: https://github.com/andreibondarev/langchainrb/issues/506#issuecomment-1992709067

Comment (emezac) [reply lib/langchain/llm/openai.rb:20]: Ok I agree

**Comment by emezac** [comment]
Sorry @andreibondarev  I was a little busy implementing a RAG application using your Gem. I was creating the implementation of the **ask** method but at the row level, I mean **ask method** at the instance level, interesting. Thanks!

**Pull request merged**